src/workerd/jsg/jsg.c++

```diff
@@ -327,6 +327,9 @@ kj::Maybe<JsObject> Lock::resolveModule(kj::StringPtr specifier) {
   auto spec = kj::Path::parse(specifier);
   auto& info = JSG_REQUIRE_NONNULL(
       moduleRegistry->resolve(*this, spec), Error, kj::str("No such module: ", specifier));
+  if (!!requireEsm) {
```

Comment:
Why do we need !!? If requireEsm has bool operator if (requireEsm) would be sufficient I believe?